package.json

```diff
@@ -4,17 +4,24 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.1",
+    "@apollo/react-testing": "^3.1.1",
     "@types/jest": "24.0.18",
     "@types/node": "12.7.8",
-    "@types/react": "16.9.3",
+    "@types/react": "^16.9.4",
     "@types/react-dom": "16.9.1",
-    "react": "16.9.0",
-    "react-dom": "16.9.0",
+    "apollo-boost": "^0.4.4",
+    "graphql": "^14.5.8",
+    "react": "^16.10.1",
+    "react-apollo": "^3.1.1",
+    "react-dom": "^16.10.1",
     "react-scripts": "3.1.2",
     "typescript": "3.6.3"
   },
   "scripts": {
     "start": "react-scripts start",
+    "server": "npm start --prefix server",
+    "postinstall": "npm install --prefix server",
```

Comment:
Is this command meant to be used by the user? If so it should be documented in readme.md

Reply:
@duiliopastorelli No, this command is auto-starting's. So whenever user runs `npm install` in the root folder, with help of the current command a deps for `/server` folder will be installed automatically as well :)